### PR TITLE
Add documentation

### PR DESCRIPTION
**Pull request by dcallao**
This PR adds the following changes:
- Updated README.md of every module, examples, and CONTRINUTING.md doc

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,95 +1,86 @@
...
-
-```hcl
-# main.tf
-terraform {
-  required_providers {
-    equinix = {
-      source = "equinix/equinix"
-    }
-}
+This repository contains a collection of Terraform modules to help automate the deployment and management of both baremetal and [Kubernetes (EKS-A)](https://github.com/equinix-labs/terraform-equinix-metal-eks-anywhere) environments running on [Equinix Metal](https://deploy.equinix.com/).
```
Comment (displague) [inline README.md:7]: EKS-A is not the focus of this repo.

Comment (displague) [reply README.md:7]: Rather it is one of the many projects that are catalogued in this repo and can be demonstrated through the workshop pattern included in this repo.

Comment (dcallao) [reply README.md]: Resolved in the latest commit. Thx.

**Comment by dcallao** [comment]
@displague This latest set of changes wraps up documentation for this module.

**Review by cprivitere — APPROVED** [review]

**Pull request merged**